node/process.go

```
@@ -5,9 +5,11 @@ import (
 	"time"
 
 	abcitypes "github.com/cometbft/cometbft/abci/types"
+	ctypes "github.com/cometbft/cometbft/rpc/core/types"
```

golangci-lint (1.62.2)

ST1019: package "github.com/cometbft/cometbft/rpc/core/types" is being imported more than once

(stylecheck)